x-pack/plugins/rollup/public/crud_app/sections/job_list/job_table/job_table.js

```diff
@@ -280,7 +279,7 @@ export class JobTableUi extends Component {
       return (
         <EuiTableRowCell
           key={`${job.id}-${name}`}
-          data-test-subj={`jobTableCell-${name}`}
+          data-test-subj={`jobTableCell-${name.replace(' ', '')}`}
```

Comment:
I know I was the one that guided you to this implementation, but I've learned a better way.  Can we try this;
`data-test-subj={`jobTableCell-${_.camelCase(name)}`}`

This change will also require a change in the rollup_page from `Indexpattern` to `IndexPattern`
and from `Rollupindex` to `RollupIndex`